### PR TITLE
Integration testing now uses webmock

### PR DESCRIPTION
**Pull request by harrisj**
## Issue #20 

Now the integration tests do not require a connection to the VBMS test server (although that is an option if necessary by setting a CONNECT environment variable)

### CODE REVIEW

File: spec/vbms_spec.rb [hunk 1]
```diff
@@ -144,7 +148,15 @@
...
-    @client = VBMS::Client.from_env_vars()
+    if ENV.key?('CONNECT')
+      # We're doing it live and connecting to VBMS test server
+      # otherwise, just use @client from above and webmock
+      @client = VBMS::Client.from_env_vars()
+    else
+      @client = VBMS::Client.new(
+        'http://test.endpoint.url/', fixture_path('test_keystore.jks'), fixture_path('test_samltoken.xml'), nil, 'importkey', nil, nil, nil
+      )
+    end
```
Comment (amoose) [inline spec/vbms_spec.rb]: What do you think about moving this to a method in spec_helper?

File: spec/vbms_spec.rb [hunk 2]
```diff
@@ -4,10 +4,14 @@
...
     @client = VBMS::Client.new(
-      nil, nil, nil, nil, nil, nil, nil
+      'http://test.endpoint.url/', nil, nil, nil, nil, nil, nil
     )
   end
 
+  after(:example) do
+    cleanup_webmock
+  end
+
```
Comment (amoose) [inline spec/vbms_spec.rb]: I'm assuming you started with this because of Timecop, but since we're not required to Timecop.return, can we remove this?

**Comment by amoose** [comment]
Looks great, @harrisj other than a couple nits. I was hoping to verify that actual integration specs were still :green_heart: but VBMS is giving me 500s today.

File: spec/vbms_spec.rb (hunk 2 above)
Comment (harrisj) [inline spec/vbms_spec.rb]: Yep, going to clean this up and a few things in the spec_helper

File: spec/vbms_spec.rb (hunk 1 above)
Comment (harrisj) [inline spec/vbms_spec.rb]: I think I want to keep this for now, but we could consider moving it down the road if used elsewhere

**Comment by amoose** [comment]
If you rebase, I can pull the trigger :gun:

**Comment by amoose** [comment]
Looks like the rebase wasn't cleanly re-written on top of the latest upstream changes. I created a new branch and rebased on top of master: `webmock-sync`

**Pull request closed without merging**